client/src/app/components/filters/filters.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+import { StateService } from 'src/app/services/state.service';
+
+@Component({
+  selector: 'app-filters',
+  templateUrl: './filters.component.html',
+  styleUrls: ['./filters.component.scss'],
+})
+export class FiltersComponent implements OnInit {
+  public topics = ['people', 'cash', 'data', 'all'];
+  public state: StateService;
+  public tagCount: any;
+
+  constructor(state: StateService) {
+    this.state = state;
+  }
+
+  ngOnInit() {
+    if (!this.state) {
+      return;
+    }
+
+    const tags = [];
+
+    this.state.activeSection.questions.forEach((question) =>
+      tags.push(...question.tags),
+    );
+
+    this.tagCount = { all: tags.length };
+
+    tags.forEach((tag) => (this.tagCount[tag] = this.tagCount[tag] + 1 || 1));
+  }
```

Comment:
To make the component a 'readable' recipe/checklist, I'd recommend to put a chunk of steps like this in a separate, named method (even it's the only thing that happens in this component)

So something like "`updateTagCount()`"